purchases/src/test/java/com/revenuecat/purchases/google/attribution/GoogleDeviceIdentifiersFetcherTests.kt

```diff
@@ -244,8 +251,9 @@ class GoogleDeviceIdentifiersFetcherTests {
             val advertisingID = identifiers[SubscriberAttributeKey.DeviceIdentifiers.GPSAdID.backendKey]
             assertThat(advertisingID).isNull()
 
-            val androidID = identifiers[SubscriberAttributeKey.DeviceIdentifiers.AndroidID.backendKey]
-            assertThat(androidID).isEqualTo("androidid")
+            // Assert we don't collect the ANDROID_ID
+            assertThat(identifiers.containsKey("\$androidId")).isFalse()
+            assertThat(identifiers.containsValue("androidid")).isFalse()
```

Comment:
Since this is repeated a few times, we can just extract it to a method I think. That way we also save the comments

Reply:
Yea I'm not opposed to that! Didn't do it at first because I saw some repetition in this test already, which might have been deliberate. It's extracted now!